### PR TITLE
Proposed changes for MAAS 2.5

### PR DESCRIPTION
**Pull request by jsseidel**

### CODE REVIEW

File: en/nodes-custom.md [hunk 1]
```diff
@@ -110,13 +128,33 @@ can both be overridden to execute the Curtin configuration in the ephemeral
 environment. Additionally, the `late` hook can be used to execute a
 configuration for a machine being installed, a state known as *in-target*.
 
+Curtin commands look like this:
```
Comment (jsseidel) [inline en/nodes-custom.md]: Note that this addresses a comment in the bug about not understanding how curtin commands work.

File: en/manage-cli-comp-hw.md [hunk 2]
```diff
@@ -193,22 +232,116 @@ included in the output:
 maas $PROFILE machine read $SYSTEM_ID
 ```
 
+## Track libvirt storage pools
+
+Retrieve pod storage pool information with the following command:
+
+```
```
Comment (degville) [inline en/manage-cli-comp-hw.md]: *bash*

File: en/manage-cli-comp-hw.md [hunk 3]
```diff
@@ -193,22 +232,116 @@ included in the output:
...
+maas $PROFILE pod read $POD_ID
+```
+
+Example:
+
+```
```
Comment (degville) [inline en/manage-cli-comp-hw.md]: We normally add *no-highlight*

File: en/nodes-comp-hw.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 # Pods
 
-Pods, or composable hardware, allow for the dynamic composition of nodes from a pool
-of available hardware resources (e.g. disk space, memory, cores).
+Pods, or composable hardware, allow for the dynamic composition of nodes from a
+pool of available hardware resources (e.g. disk space, memory, cores).
 
 This enables a machine request to be made without having machines pre-built.
-Modelling tools, such as [Juju][about-juju], can leverage this functionality
+Modeling tools, such as [Juju][about-juju], can leverage this functionality
```
Comment (degville) [inline en/nodes-comp-hw.md]: *Modelling* is the typical en-GB spelling, which we tried to keep to.

File: en/nodes-deploy.md [hunk 5]
```diff
@@ -24,6 +23,28 @@ the user's MAAS account. This is explained in
 !!! Note:
     Juju has its own way of adding SSH keys to machines under its control.
 
+MAAS also supports node customization with a process called "preseeding." For
```
Comment (degville) [inline en/nodes-deploy.md]: en-GB = customisation.

File: en/nodes-deploy.md [hunk 6]
```diff
@@ -36,11 +57,11 @@ will undergo the following process:
...
 
-!!! Note: 
+!!! Note:
     The *curtin* installer uses an image-based method and is now the only
-    installer used by MAAS. Although the older *debian-installer* method has been
-    removed curtin continues to support preseed files. See the `/etc/maas/preseed`
-    directory.
+    installer used by MAAS. Although the older *debian-installer* method has
+    been removed curtin continues to support preseed files. For more information
+    about customizing nodes see [Custom node setup][preseed].
```
Comment (degville) [inline en/nodes-deploy.md]: customising

File: en/installconfig-images.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ TODO:  bug tracking: https://github.com/canonical-webteam/documentation-builder/
 
 MAAS supports the images it generates for stable Ubuntu releases and for CentOS
 6.6 and CentOS 7. [Ubuntu Advantage][ubuntu-advantage] is needed in order to
-use Windows and RHEL images or in order to build a custom image for any
+use Windows, RHEL, and ESXi images or in order to build a custom image for any
```
[the remark this answers is not in the document]
Comment (jsseidel) [reply en/installconfig-images.md]: Ack

File: en/installconfig-upgrade.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+Title: Upgrading MAAS
+
+# Upgrading MAAS
+
+## Upgrade 2.4 to 2.5
+
+To upgrade MAAS from 2.4 to 2.5:
+
+```
```
[the remark this answers is not in the document]
Comment (jsseidel) [reply en/installconfig-upgrade.md]: ack

File: en/manage-cli-comp-hw.md (hunk 2 above)
Comment (jsseidel) [reply en/manage-cli-comp-hw.md]: ack

File: en/manage-cli-comp-hw.md (hunk 3 above)
Comment (jsseidel) [reply en/manage-cli-comp-hw.md]: ack

File: en/nodes-comp-hw.md (hunk 4 above)
Comment (jsseidel) [reply en/nodes-comp-hw.md]: ack

File: en/nodes-deploy.md (hunk 5 above)
Comment (jsseidel) [reply en/nodes-deploy.md]: Ack. Need to search and replace these . . .  and also remember to swap en_US for en_GB in vim :)

**Pull request merged**